rskj-core/src/test/java/co/rsk/core/types/bytes/BytesSliceTest.java

```diff
@@ -74,4 +113,44 @@ void testEmptySlice() {
         assertEquals(0, actualResult.length());
         assertArrayEquals(expectedResult, actualResult.copyArray());
     }
+
+    private static void checkArraycopy(Functions.Action5<Object, Integer, Object, Integer, Integer> fun) {
```

Comment:
suggestion: 

Although all variables are used here, I would suggest create local variables to hold the value of the arrays. So we can have a better understanding just by reading the code withoug have to interpret it. For example:

```
...
        byte[]  anotherDestArray= new byte[5];
        assertArrayEquals(anotherDestArray, dest); // yet unmodified

        fun.apply(origin, 0, dest, 0, 5);
        byte[] expectedDestAfterCopy = new byte[]{1,2,3,4,5};
        assertArrayEquals(expectedDestAfterCopy, dest);

        anotherDestArray = new byte[5];
        fun.apply(origin, 1, anotherDestArray, 1, 3);
        byte[] newExpectedArray = new byte[]{0,2,3,4,0};
        assertArrayEquals(newExpectedArray, anotherDestArray);
```